tutorials/using-cloud-vpn-with-strongswan/index.md

```diff
@@ -0,0 +1,534 @@
+---
+title: How to set up VPN between strongSwan and Cloud VPN
+description: Learn how to build site-to-site IPsec VPN between strongSwan and Cloud VPN.
+author: civiloid, brona
+tags: Compute Engine, Cloud VPN, Strongswan, Libreswan, firewall
+date_published: TBD
+---
+
+This guide walks you through how to configure the [strongSwan](https://www.strongswan.org/)
+for integration with the [Google Cloud VPN][cloud_vpn]. This information is
+provided as an example only. This guide is not meant to be a comprehensive
+overview of IPsec and assumes basic familiarity with the IPsec protocol.
+
+[cloud_vpn]: https://cloud.google.com/compute/docs/vpn/overview
+
+# Environment overview
+
+The equipment used in the creation of this guide is as follows:
+
+* Vendor: strongSwan
+* Software release: 5.5.1 on Debian 9.6
+
+## Topology
+
+The topology outlined by this guide is a basic site-to-site IPsec VPN tunnel
+configuration using the referenced device:
+
+![Topology](https://storage.googleapis.com/gcp-community/tutorials/using-cloud-vpn-with-strongswan/Overview.png)
+
+# Before you begin
+
+## Prerequisites
+
+To use a strongSwan with Cloud VPN make sure the following prerequisites have been met:
+
+*   VM or Server that runs strongSwan is healthy and has no known issues. 
+*   There is root access to the strongSwan instance.
+*   Your on-premises firewall allows UDP port 500, UDP port 4500, and ESP packets.
+*   You should be able to configure your on-premises router to route traffic through
+    Strongswan VPN gateway. Some environments might not give you that option.
```

Comment:
The capitalization of 'strongSwan' seems somewhat inconsistent.

Reply:
Should be consistent now. I've also fixed the same issue for "BIRD" (Bird -> BIRD).